ios/RNSScreenStackHeaderSubview.mm

```diff
@@ -72,9 +72,6 @@ - (void)updateLayoutMetrics:(const react::LayoutMetrics &)layoutMetrics
         self);
   } else {
     self.bounds = CGRect{CGPointZero, frame.size};
-    // We're forcing the parent view to layout this subview with correct frame size,
-    // see: https://github.com/software-mansion/react-native-screens/pull/2248
-    [self.superview layoutIfNeeded];
```

Comment:
I'm not super familiar with this code, but will this take away the ability to position the layout when the headerRight view changes internally (not through a `setOptions`)?

```ts
useEffect(() => {
	navigation.setOptions({
		headerRight: () => <HeaderRight />
	})
}, []) 


const HeaderRight = () => {
 const [x, setX] = useState(false)
 useTimeout(() => setX(true), 2000)
 return <View style={{backgroundColor:'green', height:20, width: x ? 40 : 20}} />
}
```

Reply:
@thomasttvo It'll be fine 😉. If you have any concerns, feel free to test this change within your project and let me know if it works as intended.